scripts/deploy-sc.sh

```diff
@@ -25,9 +25,9 @@ declare -a helmfile_opt_flags
 [[ -n "$INTERACTIVE" ]] && helmfile_opt_flags+=("$INTERACTIVE")
 
 if [ ${#} -eq 1 ] && [ "$1" = "sync" ]; then
-    helmfile -f . -e service_cluster sync
+    helmfile -f . -e service_cluster sync --concurrency 8
```

Comment:
Could you rework this into a flag so one can change this?

Reply:
Sure